apps/service-catalog/app/catalogs/[catalogId]/public-services/[serviceId]/edit/public-service-edit-page.module.css

```diff
@@ -0,0 +1,5 @@
+.heading {
+  padding-left: 13px;
```

Comment:
kanskje bruke relative enheter, som `rem`, i stedet for absolutte mål som `px`. Eks: `padding-left: 1.3rem`.